src/lib/spinel/radio_spinel_impl.hpp

```diff
@@ -1805,6 +1803,9 @@ void RadioSpinel<InterfaceType, ProcessContextType>::CalcRcpTimeOffset(void)
      */
 
     VerifyOrExit(!mIsTimeSynced || (otPlatTimeGet() >= GetNextRadioTimeRecalcStart()));
```

Comment:
Not directly related to this PR. But why do we need both `mIsTimeSynced` and `mRadioTimeRecalcStart`? 
Can we just set `mRadioTimeRecalcStart` to 0 initially and remove `mIsTimeSynced`?

Reply:
If 1.2 is not enabled, `GetNow()` should return an error number as timer is not synced.

Reply:
```c++
    mRadioTimeRecalcStart = localRxTimestamp + RCP_TIME_OFFSET_CHECK_INTERVAL;
```

Seems we only store local time in mRadioTimeRecalcStart?

Reply:
I don't get your question here. `mIsTimeSynced` is used to indicate whether the value in `mRadioTimeOffset` is valid. `mRadioTimeRecalcStart` is only used to set next resync time.

Reply:
I mean if we initialize mRadioTimeRecalcStart=0 , we can simplify the condition to:
```c++
VerifyOrExit(otPlatTimeGet() >= GetNextRadioTimeRecalcStart());
```

Reply:
Then again, in `GetNow()`, there is no variable to check if `mRadioTimeOffset` is a valid value. `mRadioTimeOffset` might not be valid because,
1. OT1.2 is not enabled.
2. Some error happens during `CalcRcpTimeOffset`

Reply:
Indeed `mIsTimeSynced` is useful in GetNow(). SGTM.